libs-ts/cover-component/package.json

```diff
@@ -11,12 +11,9 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "format": "just prettier",
-    "format:check": "just prettier-check",
-    "prepack": "bash ./prepack.sh",
-    "postpack": "bash ./postpack.sh"
+    "format:check": "just prettier-check"
   },
   "dependencies": {
-    "@friends-library/adoc-utils": "workspace:*",
```

Comment:
removing this dependency had a major effect on our bundle size, since it transitively pulled in the `hilkiah` library, which is large.